conf/db/migration/metrics_portal_ddl/postgresql/V3__create_alerts_and_expressions_table.sql

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2015 Groupon
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+CREATE TYPE context AS ENUM ('HOST', 'CLUSTER');
+CREATE TYPE operator AS ENUM ('EQUAL_TO', 'NOT_EQUAL_TO', 'LESS_THAN', 'LESS_THAN_OR_EQUAL_TO', 'GREATER_THAN', 'GREATER_THAN_OR_EQUAL_TO');
```

Comment:
So if we're going to use postgresql specific enums I would prefer we did a table create v3 with varchar, and then a specific v4 in each that changes the type. However, aside from considering not using enums (see other comment) you should also consider a domain table and a constraint which is portable:

http://stackoverflow.com/questions/9361227/how-to-replace-enum-type-in-h2-databse

@BrandonArp Please offer your advice here as well.